CHANGELOG.rst

```diff
@@ -1,9 +1,48 @@
 .. _changelog:
 
+0.45.0
+------
+
+Renku ``0.45.0`` introduces improvements when working with projects and sessions
+on both the CLI and the UI.
+
+User-Facing Changes
+~~~~~~~~~~~~~~~~~~~
+
+**✨ Improvements**
+
+- **Core Service, CLI**: Add support for specifying a project image during
+  project initialization
+  (`#3623 <https://github.com/SwissDataScienceCenter/renku-python/issues/3623>`).
```

Comment:
```suggestion
  (`#3623 <https://github.com/SwissDataScienceCenter/renku-python/issues/3623>`_).
```

Reply:
Ooops :speak_no_evil:  Thank you for spotting those!